cea/interfaces/dashboard/inputs/templates/table.html

```diff
@@ -70,9 +68,41 @@ <h3>Inputs</h3>
                 <h2>{{ table_name }}</h2>
                 <div class="clearfix"></div>
               </div>
-              <div id="mapid" class="x_content" style="height: 500px;">
-
+              <div id="style-group">
```

Comment:
i think this could be styled a bit better. looking at the template [Gentelella Alela!](https://colorlib.com/polygon/gentelella/index.html) it should probably be added in the title thing on the right with an icon. what do you think? maybe use these icons:

- [fa-cubes](https://colorlib.com/polygon/gentelella/icons.html#/cubes) for 3d
- oh, idk. maybe icons are too hard to sell... instead, you could add it to the wrench icon dropdown menu? that _does_ hide it though...

![image](https://user-images.githubusercontent.com/2969564/58562487-a2510700-8229-11e9-9a84-638bbfaa8fe3.png)